lib/frame_events.ml

```diff
@@ -114,6 +116,9 @@ end = struct
       ~low:(addr_of_address arch low)
       ~high:(addr_of_address arch high) |>
     Value.create modload
+
+  let mode (fe : string) =
+    Value.create mode (Mode.read fe) (* TODO: catch failure if string is not in enum? *)
```

Comment:
Yes, I think it would be a good idea to map unmatching values to the unknown enum value.